File: atcoder/abc397a_thermometer.cpp
// Vicfred
// https://atcoder.jp/contests/abc397/tasks/abc397_a
// implementation
#include <iostream>

using namespace std;

int main() {
  double X;
  cin >> X;
  if(X >= 38.0) {
    cout << 1 << endl;
  } else if(X >= 37.5) {
    cout << 2 << endl;
  } else {
    cout << 3 << endl;
  }
  return 0;
}

File: atcoder/abc397a_thermometer.pl
# vicfred
# https://atcoder.jp/contests/abc397/tasks/abc397_a
# implementation

# Read input
my $X = <STDIN>;
chomp($X);  # Remove newline

# Apply condition logic
if ($X >= 38.0) {
    print "1\n";
} elsif ($X >= 37.5) {
    print "2\n";
} else {
    print "3\n";
}
